src/apps/useAppWidth.js

```diff
@@ -0,0 +1,6 @@
+import { useContext } from 'react'
```

Comment:
Only place this is used is in `Permissions` no? Should we move the file into the `Permissions` folder?

Reply:
We'll need to use it in the Settings and App Center apps (and any new native apps).